### PR TITLE
Batchbook refactorings

### PR DESCRIPTION
**Pull request by ramonski**
## Description of the issue/feature this PR addresses

Code cleanup and some refactorings for batchbook (see https://github.com/senaite/senaite.core/pull/1966)
--
I confirm I have tested this PR thoroughly and coded it according to [PEP8][1]
and [Plone's Python styleguide][2] standards.

[1]: https://www.python.org/dev/peps/pep-0008
[2]: https://docs.plone.org/develop/styleguide/python.html

### CODE REVIEW

File: src/bika/lims/browser/batch/batchbook.py [hunk 1]
```diff
@@ -154,13 +167,11 @@ def folderitems(self):
             batchlink = ""
             batch = ar.getBatch()
             if batch:
-                batchlink = "<a href='%s'>%s</a>" % (
-                    batch.absolute_url(), batch.Title())
+                batchlink = get_link(api.get_url(batch), api.get_title(batch))
```
Comment (xispa) [inline src/bika/lims/browser/batch/batchbook.py]: Easier with `get_link_for`:

```python
batchlink = get_link_for(batch)
```

File: src/bika/lims/browser/batch/batchbook.py [hunk 2]
```diff
@@ -154,13 +167,11 @@ def folderitems(self):
...
 
-            arlink = "<a href='%s'>%s</a>" % (
-                ar.absolute_url(), ar.Title())
+            arlink = get_link(api.get_url(ar), api.get_title(ar))
```
Comment (xispa) [inline src/bika/lims/browser/batch/batchbook.py]:
```python
arlink = get_link_for(ar)
```

**Review by xispa — CHANGES_REQUESTED** [review]
Minor aesthetic changes!

File: src/bika/lims/browser/batch/batchbook.py (hunk 2 above)
Comment (ramonski) [reply src/bika/lims/browser/batch/batchbook.py]: Done, thanks!

**Review by xispa — APPROVED** [review]

**Pull request merged**